Loop Status Extension/StatusViewController.swift

```diff
@@ -20,33 +20,65 @@ class StatusViewController: UIViewController, NCWidgetProviding {
     @IBOutlet weak var batteryHUD: BatteryLevelHUDView!
     @IBOutlet weak var subtitleLabel: UILabel!
     
+    var context: StatusExtensionContext?
+    var defaults: UserDefaults?
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         subtitleLabel.alpha = 0
         subtitleLabel.textColor = UIColor.secondaryLabelColor
     }
     
+    override func viewWillAppear(_ animated: Bool) {
```

Comment:
this can be called multiple times, and disappear can as well. These two aren't always balanced in the case of an interactive transition.